backend/src/main/java/de/neuefische/backend/service/TimerService.java

```diff
@@ -16,8 +16,23 @@ public TimerService(TimerRepo timerRepo) {
     }
 
     public PumpTimer getTimer() {
-       return timerRepo.findFirstByOrderById().orElseThrow(
-               ()-> new IllegalStateException("No timer in database"));
+        PumpTimer pumpTimer = getTimerFromRepo();
+        if (!pumpTimer.isDone()){
```

Comment:
woudln't this be getIsDone? Or is this this special case of lombok @Data naming getteres differently when the attribute starts with 'is'. I remember something vaguely

Reply:
isDone is a boolean but I'm not sure why the getter is called isDone as well. IntelliJ suggested it.